setup.py

```diff
@@ -187,8 +187,11 @@ def run_tests(self):
     maintainer_email='jethier@google.com',
     packages=find_packages(exclude='examples'),
     package_data={'openhtf': ['output/proto/*.proto',
-                              'output/web_gui/prebuilt/**/*.*',
-                              'output/web_gui/prebuilt/*.*']},
```

Comment:
Previously we explicitly differentiated between the prebuilt frontend and the build in dist. We made frontend developers run a separate command to update the pre-built to ensure it was done intentionally. Just want to make sure the intention is to move away from that workflow.

Reply:
Will this PR deprecate old front end?

Reply:
@jettisonjoe As discussed off GitHub, I recommend that we check in `dist/` instead of `prebuilt/`. The recommended developer workflow is to use the webpack development server to develop off of `src/`, and to update `dist/` with `npm run build` when you're done making changes.

@wallacbe As discussed off GitHub, this will remove the old front end and Station API entirely. The new station server should provide at least as much functionality and be at least as performant as the StationAPI.